config.json

```diff
@@ -1,3 +1,4 @@
 {
-    "port": "25565"
+    "port": "25565",
+    "logLevel": "DEBUG"
```

Comment:
Maybe should be info by default imo

Reply:
Uh this is not the default, this is just a test config, the defaults one is Config.ts xd